src/transformers/utils/dummy_bitsandbytes_objects.py

```diff
@@ -0,0 +1,9 @@
+# This file is autogenerated by the command `make fix-copies`, do not edit.
```

Comment:
This file should be removed now.

Reply:
Fixed in [9bce8a7](https://github.com/huggingface/transformers/pull/21579/commits/9bce8a71264f013912841247e3f160c7bb769f56)